src/content/learn/index.md

```diff
@@ -119,3 +119,23 @@ Critical views of Ethereum and Cryptocurrencies.
 - [The Challenges of Building Ethereum Infrastructure](https://medium.com/@lopp/the-challenges-of-building-ethereum-infrastructure-87e443e47a4b) _Jan 8, 2018 - Jameson Lopp_
 - [Parsimonious Answers to Difficult Questions](https://www.youtube.com/watch?v=GOkSg0BuSdw&feature=youtu.be) _(Video) March 10, 2019 - Rick Dudley_
 - [There’s no good reason to trust blockchain technology](https://www.wired.com/story/theres-no-good-reason-to-trust-blockchain-technology/) _Feb 6, 2019 - Bruce Schneier_
+
+## Books and podcasts {#books-and-podcasts}
```

Comment:
I think its worthwhile throwing the Mastering Ethereum book in here as well for developers: https://github.com/ethereumbook/ethereumbook

Reply:
That's a good one! I appended it to the list of books.